Parse/Internal/Command/ParseCommand.cs

```diff
@@ -46,14 +46,13 @@ internal class ParseCommand : HttpRequest {
         IList<KeyValuePair<string, string>> headers = null,
         Stream stream = null,
         string contentType = null) {
-      Uri = new Uri(ParseClient.HostName, relativeUri);
+      Uri = new Uri(ParseClient.HostName + relativeUri);
```

Comment:
It works if both have `/`. I've tested this. But if none have `/`, it breaks.

The only issue I have with doing `new Uri(ParseClient.HostName, relativeUri);` is that

```lang=csharp
new Uri("https://api.parse.com/1/", "gogo");
```
translates into
```
https://api.parse.com/gogo
```

I'm open to better solution.